mlflow/store/model_registry/abstract_store.py

```diff
@@ -183,6 +183,7 @@ def create_model_version(
         description=None,
         local_model_path=None,
         model_id: Optional[str] = None,
+        model_params: Optional[list] = None,
```

Comment:
```suggestion
        model_params: Optional[list[<Add type here>]] = None,
```

Let's avoid an ambiguous type hint.